code/nodes/runModels/utils.q

```diff
@@ -0,0 +1,86 @@
+\d .automl
+
+// Utility functions for runmodels
+
+// @kind function
+// @category runModelsUtility
+// @fileoverview Extraction of an appropriately valued dictionary from a non complex flat file
+// @param nameMap  {sym} Name mapping to appropriate text file
+// @param filePath {str} File path relative to .automl.path
+// @return {dict} Parsed from an appropriate flat file
+runModels.i.txtParse:{[nameMap;filePath]
+  fileName:`$path,filePath,runModels.i.files nameMap;
+  runModels.i.readFile each(!).("S*";"|")0:hsym fileName
+  }
+
+// @kind function
+// @category runModelsUtility
+// @fileoverview Extraction of data from a file
+// @param filePath {str} File path from which to extract the data from 
+// @return {dict} parsed from file
+runModels.i.readFile:{[filePath]
+  key(!).("S=;")0:filePath
+  }
+
+// @kind function
+// @category runModelsUtility
+// @fileoverview Fit and score custom model to holdout set
+// @param bestModel {sym} The best scorinng model from xval
+// @param tts       {dict} Feature and target data split into training and testing set
+// @param mdls      {tab}  Models to be applied to feature data
+// @param scoreFunc {<} Scoring metric applied to evaluate the model
+// @param cfg       {dict} Configuration information assigned by the user and related to the current run
+// @return {dict} The fitted model along with the predictions
+runModels.i.customModel:{[bestModel;tts;mdls;scoreFunc;cfg]
+  if[bestModel~`multikeras;
+    tts[`ytrain`ytest]:runModels.i.prepMultiTarget tts
+    ];
+  modelDef:runModels.i.bestModelDef[mdls;bestModel]each`lib`fnc;
+  customStr:".automl.models.",sv[".";string modelDef],".";
+  model:utils.qpyFuncSearch[customStr,"model"][tts;cfg`seed];
```

Comment:
These functions can probably be get, as with above python functions aren't going to be definable within the .auto namespace